src/main/resources/META-INF/accesstransformer.cfg

```diff
@@ -382,6 +382,7 @@ public net.minecraft.world.level.block.WoodButtonBlock <init>(Lnet/minecraft/wor
 public net.minecraft.world.level.block.WoolCarpetBlock <init>(Lnet/minecraft/world/item/DyeColor;Lnet/minecraft/world/level/block/state/BlockBehaviour$Properties;)V # constructor
 #endgroup
 public net.minecraft.world.level.block.Block m_49805_(Lnet/minecraft/server/level/ServerLevel;Lnet/minecraft/core/BlockPos;I)V # popExperience
+public net.minecraft.world.level.block.RedStoneWireBlock m_55594_(Lnet/minecraft/world/level/block/state/BlockState;Lnet/minecraft/core/Direction;)Z # shouldConnectTo
```

Comment:
Please run `checkATs` as this line is in the wrong spot.

Reply:
I ran `./gradlew checkATs` in the root folder and it didnt fail. What convention is used in this file? Should I simply append the new line at the end of this file?

Reply:
Should I place it in between `#group` and `#endgroup`?

Reply:
Should I place it in between `#group` and `#endgroup`?

Reply:
Running checkATs will automatically resort the line.